README.md

```diff
@@ -134,6 +134,31 @@ httpug:
                 base_uri: 'http://google.se/'
 ```
 
+#### Authentication
+
+```yaml
+// config.yml
+httpug:
+    plugins:
+        authentication:
+            my_basic:
+                type: 'basic'
+                username: 'foo'
+                password: 'bar'
+            my_wsse:
+                type: 'wsse'
+                username: 'foo'
+                password: 'bar'
+            my_brearer:
+                type: 'bearer'
+                token: 'foo'
```

Comment:
can we use something else than "foo" here to make it visually different? maybe 'hash' ?